netbox/netbox/views/generic/bulk_views.py

```diff
@@ -321,13 +321,52 @@ def clean(self):
 
         return ImportForm(*args, **kwargs)
 
-    def _create_objects(self, form, request):
-        new_objs = []
+    def _get_records(self, form, request):
         if request.FILES:
             headers, records = form.cleaned_data['csv_file']
         else:
             headers, records = form.cleaned_data['csv']
 
+        return headers, records
+
+    def _update_objects(self, form, request, headers, records):
+        from utilities.forms import CSVModelChoiceField
+        new_objs = []
```

Comment:
Might want to rename this just to avoid confusion.
```suggestion
        updated_objs = []
```